bhuwan/src/main/java/com/lftechnology/phpjava/MyFirstClassDemo.java

```diff
@@ -0,0 +1,19 @@
+/**
+ * 
+ */
+package com.lftechnology.phpjava;
+
+/**
+ * @author bhuwan
+ *
+ */
+public class MyFirstClassDemo {
+
+    /**
+     * @param args
+     */
+    public static void main(String[] args) {
+        System.out.println("This is my first class.");
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Replace this usage of System.out or System.err by a logger. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar.java.lftechnology.com:9000/coding_rules#rule_key=squid%3AS106)